pages/my_account_page.py

```diff
@@ -0,0 +1,66 @@
+import string
+import time
+from random import choice
+from locators.locators import ConstantRegister, ConstantUser, ConstantShippingAddress
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+def create_user() -> str:
+    user = "".join(choice(string.ascii_letters) for i in range(10)) + \
+           "".join(choice(string.digits) for i in range(10))
```

Comment:
Nie kumam tej pętli. Ale jak `i` jest nie używane i chcesz pustą iteracje, to używaj `_`.
```suggestion
    user = "".join(choice(string.ascii_letters) for _ in range(10)) + \
           "".join(choice(string.digits) for _ in range(10))
```
A nie kumam bo random wybiórczy jest, nie lepiej zrobić testy po wszystkich literach z ascii_letter - to nie jest dużo znaków, a masz fajne pokrycie wszystkich przypadków https://www.geeksforgeeks.org/python-string-ascii_letters/